src/store/Configuration/mutations.ts

```diff
@@ -0,0 +1,38 @@
+import { ConfigurationStateIF } from '@/interfaces'
+import Vue from 'vue'
+import axios from '@/axios-auth'
+
+export default {
+  setConfiguration (state: ConfigurationStateIF, data: any) {
+    state.configuration = data
+  },
+  /** Use this mutator to set a specific attribute in unit testing.
+   * Like this: store.commit('setTestConfiguration', ['AUTH_WEB_URL', 'https://auth.web.url/'])
+   * */
+  setTestConfiguration (state: ConfigurationStateIF, payload: any) {
+    if (!state.configuration) {
+      Vue.set(state, 'configuration', {})
+    }
+    Vue.set(state.configuration, payload[0], payload[1])
+  },
+  setSessionVariables (state: ConfigurationStateIF, responseData: any) {
+    const hotjarId: string = responseData['HOTJAR_ID'];
+    (<any>window).hotjarId = hotjarId
```

Comment:
Looks like responseData is a dictionary?

Reply:
Does JavaScript have a dictionary type?  Would you be okay if I used an Object instead?

Reply:
Sort of. Google it.

Looks to me like dictionary/object is sort of the same thing in JS.

I commented as I was trying to understand your pattern.